var/spack/repos/builtin/packages/soapdenovo-trans/package.py

```diff
@@ -13,11 +13,14 @@ class SoapdenovoTrans(MakefilePackage):
 
     homepage = "https://github.com/aquaskyline/SOAPdenovo-Trans"
     url      = "https://github.com/aquaskyline/SOAPdenovo-Trans/archive/1.0.4.tar.gz"
+    maintainers = ['snehring']
 
     version('1.0.4', sha256='378a54cde0ebe240fb515ba67197c053cf95393645c1ae1399b3a611be2a9795')
 
     depends_on('zlib', type='link')
-    depends_on('samtools@0.1.8', type='link')
+    depends_on('samtools@0.1.8', type='link', when='target=aarch64')
```

Comment:
Do you mean:
```suggestion
    depends_on('samtools@0.1.8', type='link', when='target=aarch64:')
```
?

Reply:
Yes, thanks for catching that. I'll fix that in a bit.